fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalRelation.java

```diff
@@ -125,4 +132,7 @@ public List<Long> getSelectedPartitionIds() {
         return selectedPartitionIds;
     }
 
+    public RelationId getId() {
```

Comment:
If this method is common for both logical and physical scan, you'd better to an declaration in the `Scan` interface